yb-voyager/src/tgtdb/event.go

```diff
@@ -232,11 +231,7 @@ func getMapValuesForQuery(m map[string]*string) []interface{} {
 			values = append(values, nil)
 			continue
 		}
-		unquotedValue, err := strconv.Unquote(*value)
-		if err != nil {
-			unquotedValue = *value
-		}
-		values = append(values, unquotedValue)
+		values = append(values, value)
```

Comment:
> Why is the function returning []interface{} instead of []*string?

I think because the batch API of `pgx` for prepared statements  `ybBatch.Queue()` accepts params as `[]interface{}` in the arguments and hence all the functions in this file for params are returning `[]interface{}`.

CC: @sanyamsinghal  correct me if I am wrong here.

Reply:
yeah correct, pgx batch.Queue() API takes the params of that datatype.